lib/auth/v2/headerAuthCheck.js

```diff
@@ -15,7 +15,7 @@ function check(request, log, data) {
     timestamp = Date.parse(timestamp);
     if (!timestamp) {
         log.debug('missing security header: invalid date/timestamp');
-        return { err: errors.MissingSecurityHeader };
+        return { err: errors.AccessDenied };
```

Comment:
I would recommend adding a custom error description accompanying the error something like
```js
errors.customizeDescription('Missing Date header');
```